src/Symfony/Component/Notifier/Channel/BrowserChannel.php

```diff
@@ -22,9 +23,12 @@ final class BrowserChannel implements ChannelInterface
 {
     private RequestStack $stack;
 
-    public function __construct(RequestStack $stack)
+    private FlashMessageImportanceMapperInterface $mapper;
+
+    public function __construct(RequestStack $stack, FlashMessageImportanceMapperInterface $mapper)
```

Comment:
```suggestion
    public function __construct(RequestStack $stack, FlashMessageImportanceMapperInterface $mapper = new DefaultFlashMessageImportanceMapper())
```

To keep BC for people who are using the class directly, without Frameowkr bundle.